src/app.ts

```diff
@@ -1,13 +1,11 @@
-import express, { Request, Response } from 'express';
-import { clientError, serverError } from './middlewares';
-
-const app = express();
-
-app.get('/', (req: Request, res: Response) => {
-  res.send('Hello World!');
-});
-
-app.use(clientError);
-app.use(serverError);
-
-export default app;
+import express from 'express';
+import { clientError, serverError } from './middlewares';
+import router from './routes/therapist';
+
+const app = express();
+
+app.use('/api/v1', router);

```

Comment:
why do you use `therapist` router instead of our main router,
So please restore index file at router folder and import all subroutes there